storm-core/src/clj/org/apache/storm/daemon/executor.clj

```diff
@@ -58,7 +58,7 @@
   (.prepare grouping context (GlobalStreamId. component-id stream-id) target-tasks)
   (if (instance? LoadAwareCustomStreamGrouping grouping)
     (fn [task-id ^List values load]
-      (.chooseTasks grouping task-id values load))
+      (.chooseTasks ^LoadAwareCustomStreamGrouping grouping task-id values load))
```

Comment:
can you also give the type hint for target-tasks as List? I suspect there is no reflection warning for that, because no method is being invoked on target-tasks. but it is better to follow the practice nonetheless for safeguarding.